src/components/experiment-tracking/run-metadata/run-metadata.scss

```diff
@@ -46,55 +45,90 @@
   fill: #{$color-accent-blue};
 }
 
+// a single run when it's not in comparison mode
 .details-metadata__run {
   flex-shrink: 0;
   margin-right: 13em;
 
-  &:last-of-type {
-    margin-right: 5em;
-  }
+  max-width: 375px;
 }
 
-.details-metadata__table {
-  border-collapse: collapse;
-  max-width: 75em;
-  margin-left: -1px; // For exact alignment with content below.
-  width: 100%;
+.details-metadata__labels,
+.details-metadata__labels-comparison-view,
+.details-metadata__run--first-run,
+.details-metadata__run--first-run-comparision-view {
+  margin-right: 0;
 }
 
 .details-metadata__run td {
   font-size: 1.4em;
   padding-bottom: 16px;
   vertical-align: top;
   word-wrap: break-word; // For breaking potential long text, such as as git branch, title, etc.
+}
 
-  &:first-of-type {
-    width: 230px;
```

Comment:
I'd be careful about removing any of these widths and not adding them back where they need to be in your new markup. Things are looking a little too wide.

Your branch:

<img width="1740" alt="image" src="https://user-images.githubusercontent.com/16869061/182227499-9dd80e22-ca7e-4f3d-ab6a-f14650f171f0.png">

Demo site:

<img width="1741" alt="image" src="https://user-images.githubusercontent.com/16869061/182227569-05226bbc-ce9b-4ba1-89ff-8771c28eb215.png">